### DISCUSSION TITLE
Can you make a release?

### DISCUSSION THREAD

**Discussion by Fludem** [spatie/laravel-pdf#207] [General]
It usually makes sense to log an issue, but it currently forces the bug template.

The latest release of Laravel PDF doesn't support Browsershot v5.

The vulnerability on Browsershot is supposed to affect all versions of Browsershot before V5.0.2.

I can see in composer.json that you currently support Browsershot V5, but a release has yet to be made since this change.

Can you guys do a release that doesn't require a vulnerable version of Browsershot? I understand this implementation may not be vulnerable. It's just annoying when some of my tools complain that I have a vulnerable dependency.

**Comment by freekmurze** [comment]
done

**Reply by Fludem** [reply]
That was impressively quick haha 

Thank you!
